src/smt/expr_names.h

```diff
@@ -0,0 +1,61 @@
+/*********************                                                        */
+/*! \file expr_names.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Utility for maintaining expression names.
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef CVC4__SMT__EXPR_NAMES_H
+#define CVC4__SMT__EXPR_NAMES_H
+
+#include <string>
+
+#include "context/cdhashmap.h"
+#include "expr/node.h"
+
+namespace CVC4 {
+namespace smt {
+
+/**
+ * This utility is responsible for maintaining names for expressions.
+ */
+class ExprNames
+{
+  typedef std::unordered_map<Node, Node, NodeHashFunction> NodeToNodeHashMap;
```

Comment:
```suggestion
```
It seems this typedef is never actually used.
It we want to keep it:
```suggestion
  using NodeToNodeHashMap = std::unordered_map<Node, Node, NodeHashFunction>;
```
See [the guidelines](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#aliases) (and I very much support this)

Reply:
Yeah sorry I copied a template and forgot to clean.